helper/src/commonMain/kotlin/com/algolia/instantsearch/helper/relateditems/internal/OptionalFilter.kt

```diff
@@ -0,0 +1,9 @@
+package com.algolia.instantsearch.helper.relateditems.internal
+
+import com.algolia.instantsearch.helper.filter.state.FilterGroupID
+import com.algolia.search.model.filter.Filter
+
+internal class OptionalFilter(
```

Comment:
I think we can find a better name for this, because a tuple of filterGroupID + FilterFacets is not necessarily an OptionalFilter, it can also be pertaining to a "normal" filter
Maybe `FilterFacetAndID` or smth better